npm-packages/meteor-babel/options.js

```diff
@@ -183,18 +187,6 @@ function getDefaultsForNode8(features) {
       require("@babel/plugin-proposal-object-rest-spread")
     );
 
-    // Ensure that async functions run in a Fiber, while also taking
-    // full advantage of native async/await support in Node 8.
-    const isFiberDisabled = process.env.DISABLE_FIBERS === '1';
-    const ignoreAsyncPlugin = process.env.IGNORE_ASYNC_PLUGIN === '1';
-
-    if (!ignoreAsyncPlugin) {
-      combined.plugins.push([require("./plugins/async-await.js"), {
```

Comment:
Note that this is a breaking change for meteor-babel.

Reply:
Thinking about this more, we need to keep this since some build plugins will want to support both Meteor 2 and 3 at the same time, so they will need the async-await babel plugin when compiling for older Meteor versions. Instead of removing it, we can add an option to the features object meteor-babel uses to create the babel config.